_docs/yasnippets/README.md

```diff
@@ -0,0 +1,21 @@
+# Yasnippets Collection
+This folder contains a yasnippets collection with common templates to ease the development. Yasnippets can be integrated into Emacs: https://github.com/joaotavora/yasnippet .
+
+The snippets can also be used to provide coding guidelines and ensure a general look and feel.
+
+## Quickstart
+Copy the folders in this directory into your local yasnippets emacs configuration.
+
+If you use spacemacs they have to be placed here:
+* ` ~/.emacs.d/private/snippets`.
+
+## Usage
+Use the `key` provided in the snippet and let yasnippet expand it.
+Note: Since yasnippet uses the actual mode of the file which should be expanded ensure the mode and the snippets folder name match.
+
+### Example: Makefile Generation
+This requires the `makefile-bsdmake-mode`.
+1. Create `Makefile`
+2. Type `mdoc`
+3. Execute `yas-expand`
+4. Use `TAB` to step through the template instantiation.
```

Comment:
It was a bit hard for me to get it working. Could you maybe add a hint to the docu, that:
1. The according modes have to be installed.
2. How to install the `makefile-bsdmake-mode`
3. That either the `yas-global-mode` or `yas-minor-mode` have to be set via. `M x` or in the emacs configuration file

Reply:
Changing the folder to `makefile-gmake-mode`for better Linux support.

Reply:
Point 2 is then Ok for me.
Point 1 and 3 are obsolete. This can be read in the installation manual of yasnippet, which is already linked int the docu.